### PR TITLE
[fr] additionnal rules on comma sentence start 1

### PR DESCRIPTION
**Pull request by remymartinDev**

### CODE REVIEW

**Comment by udomai** [comment]
Note to self:

- FPs to eliminate
- put into a rulegroup VIRGULE_DEBUT_DE_PHRASE_2 (to see statistics)

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 1]
```diff
@@ -37745,6 +37745,93 @@ though, and has since been slightly modified)-->
                 <example correction=", il">Dehors<marker> il</marker> fait beau.</example>
                 <example><marker>Combien il</marker> en a pris ?</example>
             </rule>
+            <rule default="temp_off">
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:37748]: Just to resume today's call:

- [ ] please create a rulegroup named `VIRGULE_DEBUT_DE_PHRASE_2` with these rules.
→ this allows us to look at the statistics separately. Later, we will merge it with the original rule.

- [ ] please set the rulegroup `temp_off` and erase the `default="temp_off"` from the subrules

- [ ] please remove the FPs, e.g. `!`, `que` etc.

… and let me know in a comment when you're done, so we get this rule rolling 😄

**Review by udomai — CHANGES_REQUESTED** [review]

**Comment by remymartinDev** [comment]
-> 6 rules grouped in a new rulegroup
-> added [!] exception
-> also added exceptions in some rules (ex: "Certainement pas !" "Sans aucun doute possible.", etc...)

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 2]
```diff
@@ -37746,6 +37746,106 @@ though, and has since been slightly modified)-->
...
+            <rule>
+                <pattern>
+                    <token postag="SENT_START"/>
+                    <token>bien</token>
+                    <marker>
+                        <token regexp="yes">évidemment|sûr|entendu
+                            <exception scope="next" inflected="yes">que</exception>
+                        </token>
+                    </marker>
+                    <token regexp="yes">[^,‘’–)\[—!."\/;\]\|:(\…'«»“”-]+</token>
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:37759]: These collections are missing the `\?`, I'd say

**Review by udomai — APPROVED** [review]

**Pull request merged**